tests/unit/src/test/scala/tests/DefinitionSlowSuite.scala

```diff
@@ -217,6 +274,9 @@ object DefinitionSlowSuite extends BaseSlowSuite("definition") {
            |.metals/readonly/cats/Contravariant.scala:10:45: information: <error> takes no type parameters, expected: one
            |  def compose[G[_]: Contravariant]: Functor[λ[α => F[G[α]]]] =
            |                                            ^
+           |.metals/readonly/cats/Contravariant.scala:13:15: information: not found: value Contravariant
+           |      val G = Contravariant[G]
+           |              ^^^^^^^^^^^^^
```

Comment:
Hmm, I'm not sure why this change effects like this ...

Reply:
Hmm, I'm not sure either. The only thing I can think of is that we're not calling this line here in some cases https://github.com/scalameta/metals/blob/4cfc3a51daeb457a8f1e26bea4196104de517380/metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala#L1501

Either way, the change looks benign so let's keep the diff.